src/settings.rs

```diff
@@ -3,25 +3,57 @@ use std::collections::HashSet;
 
 // Describe the settings your policy expects when
 // loaded by the policy server.
-#[derive(Serialize, Deserialize, Default, Debug)]
-#[serde(default)]
+#[derive(Serialize, Deserialize, Debug, Default)]
 #[serde(rename_all = "camelCase")]
 pub(crate) struct Settings {
-    pub(crate) denied_storage_classes: HashSet<String>,
-    pub(crate) fallback_storage_class: Option<String>,
+    pub denied_storage_classes: Option<HashSet<String>>,
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
I would skip_serializing_if None for the 3 of them, it will be backwards compatible.

Reply:
Actually, we do not need this serde directive.